packages/friends/yml/en/sarah-stephenson.yml

```diff
@@ -2,15 +2,49 @@ id: 016cf5e7-fa39-4f21-8026-e2dad7592343
 name: Sarah Stephenson
 slug: sarah-stephenson
 gender: female
-description: TODO
+born: 1738
+died: 1802
+quotes:
+  - source: Sarah Stephenson (when a child)
+    text:
+      Often have I been led to make a pause, and crave of my heavenly Father, be pleased
+      to make me like unto these your servants, whatever my sufferings in this life may
+      be. If You will be with me in the way that I go, give me bread to eat and raiment to
```

Comment:
So, this got me thinking -- is this quote "modernized"? She would have said "If thou wilt be with..." right?  How do we feel about modernizing the quotations here.  Probably fine right? Have you already thought this through?